### DISCUSSION TITLE
Is there a way to show journal title and date of publication

### DISCUSSION THREAD

**Discussion by mamazebra** [asreview/asreview#1489] [Ideas]
These can be useful information for screening.  Perhaps a drug or method didn't exist before 2004, or the journal title makes clear that the scope (of that journal) won't include content I'm looking for.

**Comment by mamazebra** [comment]
aha!  I half answered my own question, can click on URL &/or DOI to see the original journal & year of publcn.

**Comment by rohitgarud** [comment]
Hi @mamazebra, One thing to note here is that the journal name is not shown while reviewing to avoid bias. Also, if you think the scope of a journal won't include content you are looking for or you want to include papers only after certain publication year, you can remove such records from the dataset using Excel functionality before you import the dataset in Asreview.. just make sure you keep track of the number of records excluded using the above mentioned criteria

**Reply by mamazebra** [reply]
yeah... but you can't make a list of every probably irrelevant journal before you start to hunt for them, it would be endless.  And it depends on combo & study design sometime how likely journal is to be irrelevant, plus someone else handed me the RIS file and it wasn't until after I started that I spotted the issue; either I lose the progress I've made so far or we all start again from scratch.   I note that RAYYAN lets you see journal & bibliog details, btw !!.  If I were that biased against journal, I'd just exclude it at the full text stage anyway for same irrational reasons.

**Reply by rohitgarud** [reply]
It makes sense.. but I meant bias toward a journal.. if it is a reputed journal, one might be tempted to accept it at the screening stage even if it is not directly aligned with the scope of the review
